classes/PodsUI.php

```diff
@@ -3769,6 +3769,12 @@ public function table( $reorder = false ) {
 			return $this->error( __( '<strong>Error:</strong> Invalid Configuration - Missing "fields" definition.', 'pods' ) );
 		}
 			?>
+			<style>
+				/* #7196 Improve tab support for /admin.php?page=pods pods table */
```

Comment:
Lets move this into `/ui/styles/src/base/_ui-list-table.scss` around line 277 where we're adjusting things within `.wp-list-table` so it can apply in other areas we use list tables with row actions too.